README.Rmd

```diff
@@ -27,162 +29,58 @@ options(max.print = 200)
-package description. It also uses the number of reverse dependencies and
-the number of downloads to weight the scores:
+Do you need to find packages solving a particular problem, e.g. "permutation test"? 
 
-```{r}
-ps("colour")[, c("score", "package", "revdeps", "downloads_last_month")]
+```{r search}
+library("pkgsearch")
+pkg_search("permutation test")
 ```
 
-### Preferring Phrases
-
-The search engine prefers matching whole phrases over single words.
-E.g. the search phrase "permutation test" will rank coin higher than
-testthat, even though testthat is a much better result for the single word
-"test". (In fact, at the time of writing testthat is not even on the first
-page of results.)
-
-```{r}
-ps("permutation test")
-```
+## Get package metadata
 
-If the whole phrase does not match, pkgsearch falls back to individual
-matching words. For example, a match from either words is enough here,
-to get on the first page of results:
+Do you want to find the dependencies the first versions of `testthat` had and when each of these versions was released? 
 
-```{r}
-ps("test http")
+```{r history}
+cran_package_history("testthat")
 ```
 
-### British vs American English
+## Discover packages
 
-The search engine uses a dictionary to make sure that package metadata
-and queries given in British and American English yield the same results.
-E.g. note the spelling of colour/color in the results:
+Do you want to know what packages are trending on CRAN these days? `pkgsearch` can help!
 
-```{r}
-ps("colour")
-ps("color")
+```{r trend}
+cran_trending()
+cran_top_downloaded()
 ```
 
-### Ascii Folding
+## Keep up with CRAN
 
-Especially when searching for package maintainer names, it is convenient
-to use the corresponding ASCII letters for non-ASCII characters in search
-phrases. E.g. the following two queries yield the same results. Note that
-case is also ignored.
+Are you curious about the latest releases or archivals?
 
 ```{r}
-ps("gabor", size = 5)
-ps("Gábor", size = 5)
+cran_events()
 ```
 
-## Get CRAN metadata
+## More info
 
-For obtaining CRAN metadata, pkgsearch uses the [crandb webservice](http://crandb.r-pkg.org).
+Find out more in [the documentation website](https://r-hub.github.io/pkgsearch/), in particular the [function reference](https://r-hub.github.io/pkgsearch/reference/index.html) and [the vignette about search features](https://r-hub.github.io/pkgsearch/articles/search.html).
 
-### Get metadata about a package
-
-```{r}
-cran_package("pkgsearch")
-cran_package("rhub")
-```
-
-### Get latest CRAN events (releases, archivals)
-
-```{r}
-cran_events()
-cran_events(limit = 5, releases = FALSE)
-cran_events(limit = 5, archivals = FALSE)
-```
+## Installation
 
-### List active (available) packages
+You can install the package from CRAN:
 
-```{r}
-cran_packages()
-cran_packages(from = "pkgsearch")
+```r
+install.packages("pkgsearch")
 ```
 
-### List R releases in the CRANDB database
+Or the development version from GitHub:
```

Comment:
I think we should not advertise the installation of development versions at all.

Reply:
But won't this README be quite confusing pre-release with all the new stuff that's not on CRAN?

Reply:
That's just a temporary thing. But then maybe we can say above to "Install the latest pkgsearch release from CRAN:" etc.